frontend/lib/intro_slides/intro_screen.dart

```diff
@@ -32,6 +32,19 @@ class IntroScreenState extends State<IntroScreen> {
         styleDescription: theme.textTheme.bodyText1?.apply(fontSizeFactor: 1.2),
       ),
     );
+    slides.add(
+      Slide(
+        title: "Wie kann ich eine Ehrenamtskarte beantragen?",
+        description: "In dem Formular geben Sie Informationen über Sie und Ihre "
```

Comment:
We can fine-tune those texts at the very end, just before publication of the updated app.